lib/weaviate/schema.rb

```diff
@@ -37,6 +38,7 @@ def create(
         req.body = {}
         req.body["class"] = class_name
         req.body["description"] = description unless description.nil?
+        req.body["multiTenancyConfig"] = multiTenancyConfig unless multiTenancyConfig.nil?
```

Comment:
@obie Mind changing this to `multi_tenancy_config` to follow the pattern with other params here?